### ISSUE TITLE
Japanese translation of the TS2637 error message is wrong.

### ISSUE THREAD

**Issue by Masa-Shin** [microsoft/TypeScript#54079]
# Bug Report

### 🔎 Search Terms

translation Japanese variance TS2637

### 🕗 Version & Regression Information

- This is the behavior in every version I tried.

### ⏯ Playground Link

[Playground link with relevant code](https://www.typescriptlang.org/play?#code/C4TwDgpgBAkgJhAdsAlqAPCxUAqA+KAXlyA)

Error messages in the playground are not localized, therefore I have attached an image taken in VSCode:

<img width="860" alt="スクリーンショット 2023-05-01 11 55 38" src="https://user-images.githubusercontent.com/34234442/235395214-11a6f9c9-010e-4437-95eb-2cf6b411f4a3.png">

### 💻 Code

[Here is the proposed change.](https://github.com/microsoft/TypeScript/commit/aac70d75e7afdcd071d2ba8fe7050de18305dce3)

### 🙁 Actual behavior

The term 'variance' is translated to '分散', which is a statistical term that refers to a measure of how much a set of values differs from their mean.

### 🙂 Expected behavior

The term 'variance' is translated to '変性', which is a correct term for type variance.

**Comment by nmain** [comment]
Does this message also need to be changed?  https://github.com/microsoft/TypeScript/blob/main/src/loc/lcl/jpn/diagnosticMessages/diagnosticMessages.generated.json.lcl#L14352

**Comment by Masa-Shin** [comment]
@nmain 
Yes, the word '差異' just means 'difference' so this also needs to be fixed.

**Comment by Masa-Shin** [comment]
Thanks I made [a commit](https://github.com/Masa-Shin/TypeScript/commit/3d82017257240cde7c62751dbc9b4e4f7ceaff8d)for that part as well.

**Comment by Masa-Shin** [comment]
Can I make a PR for this?
